docker-compose.debug.yml

```diff
@@ -0,0 +1,46 @@
+# 5500 - Frontend
+# 5501 - Backend
+# 5502 - postgres
+# 5503 - redis
+
+# You can initialize the database with data using the "Docker.initdb" image
+# Make sure when creating the container that you specify the --network dofuslab_default switch
+
+version: '3.4'
+
+services:
+  redis:
+    image: "redis:alpine"
+    ports:
+      - "5503:6379"
+
+  postgres:
+    image: "postgres:alpine"
```

Comment:
Would personally pin to versions for both redis/postgres.

Reply:
that is a good suggestion